### PR TITLE
Metaquantome html image output

### PR DESCRIPTION
**Pull request by pravs3683**
Generates both png and HTML output. Image in HTML is viewed at 600 by 600 pixels.

### CODE REVIEW

File: tools/metaquantome/metaquantome_viz.xml [hunk 1]
```diff
@@ -90,7 +90,36 @@
...
+            <img src=\"outfile.png\" width=600 height=600>
+            <br>
+            <p>To download the image, right-click and choose \"Save image as...\"</p>
+            <br>
+            <p>OR</p>
+            <br>
+            <p><a href=\"outfile.png\">Right Click Here</a></p>
+          </body>
+        </html>" > wrapped_outfile.html &&
+        cat wrapped_outfile.html
```
Comment (bgruening) [inline tools/metaquantome/metaquantome_viz.xml]: is that cat still needed?

Comment (pravs3683) [reply tools/metaquantome/metaquantome_viz.xml]: Thanks for pointing this out. "cat" not required.

Comment (pravs3683) [reply tools/metaquantome/metaquantome_viz.xml]: Updated the code.

File: tools/metaquantome/metaquantome_db.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        mkdir -p data &&
+        metaquantome db $databases
+        --dir data
+        &&
+        tar --create --file $outfile data
```
Comment (bgruening) [inline tools/metaquantome/metaquantome_db.xml]: quotes

File: tools/metaquantome/metaquantome_db.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+<tool id="metaquantome_db" name="metaQuantome: download " version="@GVERSION@">
```
Comment (bgruening) [inline tools/metaquantome/metaquantome_db.xml]: metaQuantome database download

File: tools/metaquantome/macros.xml [hunk 4]
```diff
@@ -1,10 +1,9 @@
 <macros>
-    <token name="@VERSION@">0.99.3</token>
-    <token name="@GVERSION@">0.99.3.0</token>
+    <token name="@VERSION@">0.99.4a0</token>
+    <token name="@GVERSION@">0.99.4a00</token>
```
Comment (bgruening) [inline tools/metaquantome/macros.xml]: 4a0.0?

**Comment by caleb-easterly** [comment]
closing this - most recent in #357

File: tools/metaquantome/macros.xml (hunk 4 above)
Comment (caleb-easterly) [reply tools/metaquantome/macros.xml]: that's not allowed by `planemo lint`, so i changed it to `0.99.4a0-0`. It'll get simpler once we test - and we'll hopefully release a v 1.0!

**Pull request closed without merging**